.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.npm
+          **/node_modules
+        key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-npm-
+          ${{ runner.os }}-
+
+    - name: Read .nvmrc
+      run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+      shell: bash
+      id: nvm
+
+    - name: Install ffmpeg for firefox
+      run: sudo apt-get install ffmpeg pulseaudio
+      if: ${{ startsWith(matrix.os, 'ubuntu') }}
+
+    - name: start pulseaudio for firefox
+      run: pulseaudio -D
+      if: ${{ startsWith(matrix.os, 'ubuntu') }}
+
+    - name: Setup node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '${{ steps.nvm.outputs.NVMRC }}'
+    - run: npm i --prefer-offline --no-audit
+
+    # run safari, chrome, firefox on macos
+    - name: Run Mac test
+      run: npm run test
+      if: ${{ startsWith(matrix.os, 'macos') }}
+
+    # only run ie 11 on windows
+    - name: Run Windows test
+      run: npm run test
+      if: ${{ startsWith(matrix.os, 'windows') }}
+
+    # run chrome/firefox in linux
```

Comment:
is it possible to run these in parallel?
Though, we can do that as a followup too.

Reply:
Though, currently we're only running on linux for BS?
I'm OK with making these updates as a follow-up.

Reply:
all os steps run in parallel yeah, we are only running bs on linux

Reply:
We are actually only running on linux right now period for everything, as windows/mac are not included in the os matrix list at all.

Reply:
cool, yeah, wasn't sure whether the matrix items run in parallel or not.